src/alaveteli.yaml

```diff
@@ -0,0 +1,36 @@
+---
+description: >
+  Platform for making public freedom of information requests to public bodies.
+  Alaveteli helps you lower the barriers that prevent citizens asking questions
+  of those in power.
```

Comment:
How about adding something like `Avaveteli has been [implemented for 24 different countries]{https://alaveteli.org/deployments/}`?